### PR TITLE
Fix typos and grammar

### PR DESCRIPTION
**Pull request by benwicks**

### CODE REVIEW

**Comment by JakeWharton** [comment]
Thanks!

File: style.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ private val defaultCharset: Charset? =
         EncodingRegistry.getInstance().getDefaultCharsetForPropertiesFiles(file)
 ```
 
-Properties declaring a `get` and/or `set` function should place each on their only line with a normal indent (+4). Format them using the same rules as functions.
+Properties declaring a `get` and/or `set` function should place each on their own line with a normal indent (+4). Format them using the same rules as functions.
```
Comment (denis778) [inline style.md]: +4

File: style.md [hunk 2]
```diff
@@ -637,7 +637,7 @@ val EMPTY_ARRAY = arrayOf<SomeMutableType>()
 
 These names are typically nouns or noun phrases.
 
-Constant values can only be defined in side of an `object` or as a top-level declaration. Values otherwise meeting the requirement of a constant but defined inside of a `class` must use a non-constant name.
+Constant values can only be defined inside of an `object` or as a top-level declaration. Values otherwise meeting the requirement of a constant but defined inside of a `class` must use a non-constant name.
 
 Constants which are scalar values must use the [`const` modifier](http://kotlinlang.org/docs/reference/properties.html#compile-time-constants).
```
Comment (denis778) [inline style.md]: 7da4316

Comment (denis778) [reply style.md]: 640

Comment (denis778) [reply style.md]: 640

**Pull request merged**